tools/gltfViewer/main.qml

```diff
@@ -0,0 +1,97 @@
+import Qt3D.Core 2.12
+import Qt3D.Render 2.12
+import Qt3D.Animation 2.12
+import Qt3D.Input 2.12
+import Qt3D.Extras 2.12
+import Kuesa 1.0 as Kuesa
+import KuesaViewer 1.0 as KuesaViewer
+
+Kuesa.SceneEntity {
+    id: root
+
+    property url source: _gltfFile
+    onSourceChanged: {
+        root.clearCollections()
+        importer.source = source
+    }
+
+    Entity {
+        parent: frameGraph.camera
+        components: [
+            PointLight {
+                constantAttenuation: 1.0
+                linearAttenuation: 0.0
+                quadraticAttenuation: 0.0
+            }
+        ]
+    }
+
+    components: [
+        RenderSettings {
+            activeFrameGraph: Kuesa.ForwardRenderer {
+                id: frameGraph
+                camera: cameraAsset.node ? cameraAsset.node : fallbackCamera
+                clearColor: "white"
+            }
+        },
+        InputSettings { },
+        EnvironmentLight {
+            id: envLight
+            property string envMapFormat: Qt.platform.os == "osx" ? "_16f" : ""
+
+            irradiance: TextureLoader {
+                source: "qrc:/wobbly_bridge" + envLight.envMapFormat + "_irradiance.dds"
+                wrapMode {
+                    x: WrapMode.ClampToEdge
+                    y: WrapMode.ClampToEdge
+                }
+                generateMipMaps: false
+            }
+            specular: TextureLoader {
+                source: "qrc:/wobbly_bridge" + envLight.envMapFormat + "_specular.dds"
+                wrapMode {
+                    x: WrapMode.ClampToEdge
+                    y: WrapMode.ClampToEdge
+                }
+                generateMipMaps: false
+            }
+        }
+    ]
+
+    Camera {
+        id: fallbackCamera
+        objectName: "fallbackCamera"
+
+        position: Qt.vector3d(0.0, 0.0, 7.0)
+        upVector: Qt.vector3d(0.0, 1.0, 0.0)
+        viewCenter: Qt.vector3d(0.0, 0.0, 0.0)
+        aspectRatio: _winSize.width / _winSize.height
+    }
+
+    KuesaViewer.OrbitCameraController {
+        id: controller
+        camera: frameGraph.camera
+        windowSize: _winSize
+    }
+
+    Kuesa.Asset {
+        id: cameraAsset
+        name: root.cameras.names.length ? root.cameras.names[0] : ""
```

Comment:
What happens if we have several cameras? Maybe a key to change between cameras would be nice. Also a key to play/stop all animations?

Reply:
could use tab to cycle through cameras.
Animations is also possible maybe later. Want to keep this as simple as possible, mostly to use to test on various hardware and with various files.